.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+
+  # Maintain dependencies for go
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    labels:
+      - "ok-to-test"
+    ignore:
+      # Ignore patch versions for dependencies updates
+      - update-types: ["version-update:semver-patch"]
```

Comment:
rather, we want to ignore minor an major

Reply:
Added.